docs/core/plugins/sieve.md

```diff
@@ -194,34 +269,36 @@ binary containing the updated data ID.
 
 #### Configuration
 
-The script location syntax is specified as follows:
-
-```
-sieve = dict:<dict-uri>[;<option>[=<value>][;...]]
-```
-The `<dict-uri>` path is a Dovecot dict uri.
-
-If the name of the Script is left unspecified and is not otherwise
-provided by the Sieve interpreter, the name defaults to `default`.
+The `dict` storage driver supports all settings described in
+[Common Settings](#common-settings).
 
-##### Settings
+##### `sieve_script_name`
 
-This location supports all settings described in
-[Common Settings](#common-settings).
+If the name of the Script is left unspecified and not otherwise provided by the
+Sieve interpreter, the name defaults to `default`.
```

Comment:
How would the script name be provided by sieve interpreter? Also I'm not sure if this is saying that the behavior is different depending on whether sieve_script_name=default is explicitly set vs. when leaving it unset. I guess I don't understand what this sentence is saying at all.

Is it even important enough to mention this setting at all here? If yes, should also somehow link to the setting.